### DISCUSSION TITLE
Clase de programación II

### DISCUSSION THREAD

**Discussion by andreslb9** [mmasias/23-24-prg2#20] [General]
Hola, queríamos preguntarte si mañana al final hay clase de programación II.
Un saludo.

**Comment by mmasias** [comment]
Hola @andreslb9  

Como les comenté en clase mañana tengo una reunión que probablemente se extienda, por lo que no tendremos sesión de Programación II. Continuamos el viernes 16: aprovechar para terminar bien el ejercicio que está planteado. Mirar de dejar adecuadamente documentado el proceso de creación, tal como lo vimos en la clase anterior. Eso (incluso más que el código) es lo clave para este reto: he ido viendo algunos, voy dejando comentarios.

Un saludo
